Submodels/MTSOL_call.py

```
@@ -1,20 +1,613 @@
+                               iter_count,
+                               )
+        elif (exit_flag == ExitFlag.COMPLETED.value or 
+              exit_flag == ExitFlag.SUCCESS.value or 
+              exit_flag == ExitFlag.NOT_PERFORMED.value):
+            return
+        else:
+            raise OSError(f"Unknown exit flag {exit_flag} encountered!") from None
+    
+
+    def caller(self,
+               Run_viscous: bool = False,
```

Ruff (0.8.2)

Boolean-typed positional argument in function definition

(FBT001)